src/components/popover/popover.ts

```diff
@@ -81,6 +81,10 @@ export class Popover extends Component<PopoverProps, SpreadsheetChildEnv> {
       this.currentDisplayValue = newDisplay;
 
       if (!anchor) return;
+      el.style.top = "";
+      el.style.left = "";
+      el.style["max-height"] = "";
+      el.style["max-width"] = "";
```

Comment:
you're resetting this here to have the correct `getBoundingClientRect` just below ? Is that right ?

Reply:
yes